app/views/items/_form.html.erb

```diff
@@ -8,7 +8,12 @@
 
     <%= f.input :category, label: "Category", wrapper: :vertical_input_group do %>
       <span class="input-group-addon"><i class="fa fa-cubes"></i></span>
-      <%= f.input_field :category, class: "form-control" %>
+      <%= f.input_field :category, class: "form-control", list:"category" %>
+      <datalist id="category">
+        <%- Item.all.map(&:category).uniq.each do |category| %>
```

Comment:
I think this needs to be CanonicalItem not Item.

Sorry reviews are moving slow, there is generally a bit of a dip in activity after the event and I know the team team leads for this project have a lot of real life stuff going on right now... In fact, one of them just got married! :)

Reply:
Even though this is for the item views? (new/update)

Reply:
I think so. All items are a type of canonical item. Let me take a closer look :)

Reply:
Ignore my previous comments. :)